parent/pom.xml

```diff
@@ -108,6 +108,7 @@
         <maven.bundle.plugin.version>3.5.0</maven.bundle.plugin.version>
         <maven.checkstyle.plugin.version>3.1.1</maven.checkstyle.plugin.version>
         <checkstyle.puppycrawl.version>8.35</checkstyle.puppycrawl.version>
+        <checkstyle.skip>true</checkstyle.skip>
```

Comment:
Why this placed under the `Dependency versions` section?

Reply:
+1, move it to plugin.configuration block

Reply:
Agree, fixed.